src/css/archwizard.scss

```diff
@@ -1,9 +1,97 @@
-// color definitions
-$wz-color-default: #E6E6E6 !default;
-$wz-color-current: #808080 !default;
-$wz-color-done: #339933 !default;
-$wz-color-optional: #38ef38 !default;
-$wz-color-editing: #FF0000 !default;
+// Wizard layouts to be generated (all by default).
+// Override this variable to exclude layouts you don't need.
+$wz-layouts: 'horizontal' 'vertical' !default;
+
+// Wizard indicator styles to be generated (all by default).
+// Override this variable to exclude indicator styles you don't use.
+$wz-indicator-styles: 'small', 'large-filled', 'large-empty', 'large-filled-symbols', 'large-empty-symbols' !default;
+
+// Color definitions

```

Comment:
Does it maybe make sense to use a different comment style here? I think something like:

```scss
/**
 * ...
 * ...
 */
```

may be easier to read, but I don't know any conventions for comments in scss

Reply:
Hmm... It looks like SCSS removes `//` but preserves `/**...*/`-style comments in output CSS, probably because only `/**...*/` ones are valid CSS.

In most cases these comments are irrelevant in CSS because they document SCSS variables, functions and mixins which are no longer there.

And SCSS compiler provides no option to drop comments: https://github.com/sass/node-sass/issues/1567.

I think we'd better stay with `//`

Reply:
I agree